### PR TITLE
Enable service links

### PR DESCRIPTION
**Pull request by blockjesper**
Add support to disable service links. #104

### CODE REVIEW

File: fiaas_deploy_daemon/config.py [hunk 1]
```diff
@@ -328,6 +328,15 @@ def _parse_args(self, args):
             default="1",
             type=_int_or_unicode
         )
+        parser.add_argument(
+            "--disable-service-links",
+            dest="enable_service_links",
```
Comment (herodes1991) [inline fiaas_deploy_daemon/config.py:333]: For me, this is confusing 😅.
If the flag is --disable-service-links, the config should be the same 😄

Comment (blockjesper) [reply fiaas_deploy_daemon/config.py:333]: I agree, but I think this is the reasonable way of doing it due to some unfortunate circumstances.

1) The kubernetes API takes the argument as enableServiceLinks, with default for now being true.
2) The library used for parsing command line arguments doesn't take an argument for boolean arguments, so it's not possible to send in false together with it. My initial attempt was with the argument as "enable-service-links", but hit this problem. 

You do definitely have a point that it's confusing however.

Comment (henrik242) [reply fiaas_deploy_daemon/config.py:333]: Maybe just add a comment explaining this?

Comment (blockjesper) [reply fiaas_deploy_daemon/config.py:334]: @henrik242 I've added a comment about the inverted logic.

**Pull request merged**